terraform/vpc.tf

```diff
@@ -8,8 +8,8 @@ module "vpc" {
   name                 = "vpc"
   cidr                 = "10.0.0.0/16"
   azs                  = data.aws_availability_zones.available.names
-  private_subnets      = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
-  public_subnets       = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+  private_subnets      = ["10.0.0.0/20", "10.0.16.0/20", "10.0.32.0/20"]
```

Comment:
Can we autogen these instead of manually specifying the nets? That way we can just increase iterations on a for loop instead of having to know how CIDRs work.

Can just loop over this and increase the `netnum` param i think:
https://www.terraform.io/language/functions/cidrsubnet

Reply:
Per offline discussion, we're not going to do this because it obscures the actual CIDRs generated and theoretically CIDRs in our VPC are very sticky because changing them requires deleting most of the resources in our AWS account.